src/content/docs/dropins/product-details/pdp-containers.mdx

```diff
@@ -149,7 +147,7 @@ The following example demonstrates how to configure the Product Details containe
 return productRenderer.render(ProductDetails, {
   sku: yourGetSkuFetchFunction(),
   hideSku: false,
-  productData: {
+  initialData: {
```

Comment:
let's leave this as productData for now and I can take a closer look after and see how it would be best to document this and the initialData